course_info/views.py

```diff
@@ -25,6 +26,7 @@ def index(request):
     # to the lti launch's user.
     context = {
         'schools': _get_schools_context(canvas_user_id),
+        'canvas_roles': _get_canvas_roles(),
```

Comment:
Ditto for `UserRole` vs Canvas role question -- I think that our app uses `UserRole` ids, not Canvas role IDs. So we have to do a mapping using the `UserRole.canvas_role_id` property to connect the right labels to the right `UserRole` ids.